README.md

```diff
@@ -2,15 +2,47 @@
 
 # Prometheus rules chart
 
-**What is this app?**
-
-This repository contains Giant Swarm alerting and recording rules
-
-### Alerting
+## Table of Contents
```

Comment:
That's nice to have it included in the readme, but do you know that github already automatically generates these?